File: ContadorVocalesConsonantes.c
#include <stdio.h>
#include <ctype.h>
#include <string.h>

int main() {
    char palabra[100];
    int vocales = 0, consonantes = 0;
    
    printf("Ingrese una palabra: ");
    fgets(palabra, sizeof(palabra), stdin);
    
    for (int i = 0; i < strlen(palabra); i++) {
        char letra = tolower(palabra[i]);
        
        if (letra >= 'a' && letra <= 'z') {
            if (letra == 'a' || letra == 'e' || letra == 'i' || 
                letra == 'o' || letra == 'u') {
                vocales++;
            } else {
                consonantes++;
            }
        }
    }
    
    printf("Vocales: %d\n", vocales);
    printf("Consonantes: %d\n", consonantes);
    
    return 0;
}

File: ContadorVocalesConsonantes.java
import java.util.Scanner;

public class ContadorVocalesConsonantes {
    public static void main(String[] args) {

        System.out.print("Ingrese una palabra: ");

        Scanner scanner = new Scanner(System.in);
        String palabra = scanner.nextLine().toLowerCase();
        
        int vocales = 0;
        int consonantes = 0;
        
        for (int i = 0; i < palabra.length(); i++) {
            char letra = palabra.charAt(i);
            
            if (letra >= 'a' && letra <= 'z') {
                
                if (letra == 'a' || letra == 'e' || letra == 'i' || 
                    letra == 'o' || letra == 'u') {
                    vocales++;
                } else {
                    consonantes++;
                }
            }
        }
        
        System.out.println("Vocales: " + vocales);
        System.out.println("Consonantes: " + consonantes);
        
        scanner.close();
    }
}